src/test/components/TimelineMarkers.test.js

```diff
@@ -148,4 +229,95 @@ describe('TimelineMarkers', function() {
 
     delete window.devicePixelRatio;
   });
+
+  describe('displays context menus', () => {
```

Comment:
Those new tests that I added are the same as for the context menu in MarkerChart. This required moving a bunch of stuff around though, this is why diff is so big 😓